src/renderer/components/+events/events.tsx

```diff
@@ -108,7 +108,7 @@ export class Events extends React.Component<Props> {
           { title: "Involved Object", className: "object", sortBy: columnId.object, id: columnId.object },
           { title: "Source", className: "source", id: columnId.source },
           { title: "Count", className: "count", sortBy: columnId.count, id: columnId.count },
-          { title: "Age", className: "age", sortBy: columnId.age, id: columnId.age },
+          { title: "Last Seen", className: "age", sortBy: columnId.age, id: columnId.age },
```

Comment:
Line:81 might be broken since `creationTimestamp` returns a iso-data string, not timestamp (number)

```ts
[columnId.age]: (event: KubeEvent) => event.metadata.creationTimestamp,
```